cmd/state-tray/internal/open/open_lin.go

```diff
@@ -2,12 +2,29 @@
 
 package open
 
-import "errors"
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/ActiveState/cli/internal/locale"
+)
 
 func Prompt(command string) error {
-	return errors.New("not implemented")
-}
+	shellData, err := exec.Command("bash", "-c",
+		"grep ^$(id -un): /etc/passwd | cut -d: -f7-",
+	).Output()
```

Comment:
This is the Ubuntu "default" approach to get the user . Or do you mean that we'd just make the user always use Bash for terminals we pull up for them?

This also works with "sh" instead of "bash", but these deps are very much expected to be in any Ubuntu install (bash, grep, cut).

Reply:
At the very least use Go to do this. Read /etc/passwd and parse the part you're interested in. There's no reason for you to be invoking bash commands like this.